services/minespace-web/src/components/Forms/projects/projectSummary/Applicant.tsx

```diff
@@ -21,6 +21,8 @@ import { getDropdownProvinceOptions } from "@mds/common/redux/selectors/staticCo
 import RenderSelect from "@mds/common/components/forms/RenderSelect";
 import RenderCheckbox from "@mds/common/components/forms/RenderCheckbox";
 import { normalizePhone } from "@common/utils/helpers";
+import { fetchOrgBookCredential } from "@mds/common/redux/actionCreators/orgbookActionCreator";
```

Comment:
SonarCloud wants this import to be combined with the one on line 18.

Reply:
Updated.